server/middleware/fetchFrontendComponentMiddleware.ts

```diff
@@ -0,0 +1,28 @@
+import { Router } from 'express'
+import { DataAccess } from '../data'
+import logger from '../../logger'
+import config from '../config'
+
+export default function setUpFrontendComponents({ frontendComponentApiClient }: DataAccess): Router {
+  const router = Router({ mergeParams: true })
```

Comment:
What does `mergeParams` do? Why are we setting it?

Reply:
Removed as part of rewrite